src/extensions/animation/index.js

```diff
@@ -13,60 +13,51 @@ import classnames from 'classnames';
 /**
  * WordPress Dependencies
  */
-import { useState, useEffect, useRef } from '@wordpress/element';
-import { withSelect } from '@wordpress/data';
-import { addFilter } from '@wordpress/hooks';
-import { compose, createHigherOrderComponent } from '@wordpress/compose';
+import { select } from '@wordpress/data';
+import { useEffect, useRef, useState } from '@wordpress/element';
 
 const allowedBlocks = [
-	{ blockType: 'coblocks/gallery-carousel', animateChildren: false },
-	{ blockType: 'coblocks/gallery-collage', animateChildren: true },
-	{ blockType: 'coblocks/gallery-masonry', animateChildren: true },
-	{ blockType: 'coblocks/gallery-offset', animateChildren: true },
-	{ blockType: 'coblocks/gallery-stacked', animateChildren: true },
-	{ blockType: 'core/columns', animateChildren: false },
-	{ blockType: 'core/cover', animateChildren: false },
-	{ blockType: 'core/gallery', animateChildren: false },
-	{ blockType: 'core/group', animateChildren: false },
-	{ blockType: 'core/image', animateChildren: false },
-	{ blockType: 'core/media-text', animateChildren: false },
+	{ animateChildren: false, blockType: 'coblocks/gallery-carousel' },
+	{ animateChildren: true, blockType: 'coblocks/gallery-collage' },
+	{ animateChildren: false, blockType: 'coblocks/gallery-masonry' },
+	{ animateChildren: true, blockType: 'coblocks/gallery-offset' },
+	{ animateChildren: true, blockType: 'coblocks/gallery-stacked' },
+	{ animateChildren: false, blockType: 'core/columns' },
+	{ animateChildren: false, blockType: 'core/cover' },
+	{ animateChildren: false, blockType: 'core/gallery' },
+	{ animateChildren: false, blockType: 'core/group' },
+	{ animateChildren: false, blockType: 'core/image' },
+	{ animateChildren: false, blockType: 'core/media-text' },
 ];
 
 const animateClass = 'coblocks-animate';
 const dataAnimationHolder = 'data-coblocks-animation';
 
 /**
- * Override the default edit UI to include a new block toolbar control
+ * React hook function to extend the block editor with custom controls.
  *
- * @param {Function} BlockEdit Original component.
- * @return {string} Wrapped component.
+ * @function useAnimationControls
+ * @param {Object} props
+ * @return {JSX} Wrapped component.
  */
-const withControls = createHigherOrderComponent( ( BlockEdit ) => {
-	return enhance( ( { select, ...props } ) => {
-		const block = select( 'core/block-editor' ).getBlock( props.clientId );
-
-		return (
-			<>
-				<BlockEdit { ...props } />
-				{ props.isSelected && allowedBlocks.some( ( allowedBlock ) => allowedBlock.blockType === props.name ) && <Controls { ...{ ...props, selectedBlock: block } } /> }
-			</>
-		);
-	} );
-}, 'withControls' );
-
-addFilter(
-	'editor.BlockEdit',
-	'coblocks/animations',
-	withControls
-);
+const useAnimationControls = ( ( props ) => {
+	const block = select( 'core/block-editor' ).getBlock( props.clientId );
+
+	return (
+		<>
```

Comment:
@aledesma-godaddy can you check if the fragment is even required here?

Reply:
Tested this again. Without the JSX fragments, I am receiving parsing errors with unexpected tokens. I think it might be because it's a conditional wrap meant to be within JSX as opposed to a return null early or something like that.